src/tri/_tri.h

```diff
@@ -790,29 +790,3 @@ class TrapezoidMapTriFinder
 
     Node* _tree;    // Root node of the trapezoid map search tree.  Owned.
 };
-
-
-
-/* Linear congruential random number generator.  Edges in the triangulation are
- * randomly shuffled before being added to the trapezoid map.  Want the
- * shuffling to be identical across different operating systems and the same
- * regardless of previous random number use.  Would prefer to use a STL or
- * Boost random number generator, but support is not consistent across
- * different operating systems so implementing own here.
- *
- * This is not particularly random, but is perfectly adequate for the use here.
- * Coefficients taken from Numerical Recipes in C. */
-class RandomNumberGenerator
-{
-public:
-    RandomNumberGenerator(unsigned long seed);
-
-    // Return random integer in the range 0 to max_value-1.
-    unsigned long operator()(unsigned long max_value);
-
-private:
-    const unsigned long _m, _a, _c;
-    unsigned long _seed;
-};
-
-#endif
```

Comment:
This line needs to be kept.

Reply:
I took the liberty to add the line back again.